runtime/permissions.rs

```diff
@@ -942,6 +947,24 @@ mod tests {
     }
   }
 
+  #[test]
+  fn check_invalid_specifiers() {
+    let perms = Permissions::allow_all();
+
+    let mut test_cases = vec![];
+
+    if cfg!(target_os = "windows") {
```

Comment:
can this be removed? this test doesn't check anything on non-windows.

Reply:
@ry added a non-Windows test case (a local-file URI which is invalid on UNIX)